pkg/federation/sdlmerge/sdlmerge_test.go

```diff
@@ -618,26 +619,46 @@ const (
 	`
 )
 
-func NonIdenticalSharedTypeMergeErrorMessage(typeName string) string {
+func nonIdenticalSharedTypeMergeErrorMessage(typeName string) string {
 	return fmt.Sprintf("merge ast: walk: external: the shared type named '%s' must be identical in any subgraphs to federate, locations: [], path: []", typeName)
 }
 
-func DuplicateEntityMergeErrorMessage(typeName string) string {
+func duplicateEntityMergeErrorMessage(typeName string) string {
 	return fmt.Sprintf("merge ast: walk: external: entities must not be shared types, but the entity named '%s' is duplicated in other subgraph(s), locations: [], path: []", typeName)
 }
 
-func SharedTypeExtensionErrorMessage(typeName string) string {
+func sharedTypeExtensionErrorMessage(typeName string) string {
 	return fmt.Sprintf("the type named '%s' cannot be extended because it is a shared type", typeName)
 }
 
-func EmptyTypeBodyErrorMessage(definitionType, typeName string) string {
+func emptyTypeBodyErrorMessage(definitionType, typeName string) string {
 	return fmt.Sprintf("validate schema: external: the %s named '%s' is invalid due to an empty body, locations: [], path: []", definitionType, typeName)
 }
 
-func UnresolvedExtensionOrphansErrorMessage(typeName string) string {
+func unresolvedExtensionOrphansErrorMessage(typeName string) string {
 	return fmt.Sprintf("the extension orphan named '%s' was never resolved in the supergraph", typeName)
 }
 
-func UnresolvedExtensionOrphansMergeErrorMessage(typeName string) string {
+func unresolvedExtensionOrphansMergeErrorMessage(typeName string) string {
 	return fmt.Sprintf("merge ast: walk: external: the extension orphan named '%s' was never resolved in the supergraph, locations: [], path: []", typeName)
 }
+
+func externalDirectiveErrorMessage(typeName string) string {
```

Comment:
do we need to have duplicates of external errors here?

Reply:
This is just to match the error in the tests, because I cannot retrieve just the message part.